opennms-doc/guide-admin/src/asciidoc/text/compatibility.adoc

```diff
@@ -0,0 +1,18 @@
+[[compatibility]]
+== Compatibility
+
+{opennms-product-name} {opennms-version} requires the following component versions:
+
+[options="header, autowidth"]
+|===
+| Component               | Version Compatibility
+| OpenNMS Integration API | 0.2.1
+| Java Development Kit    | OpenJDK 8, OpenJDK 11
+| PostgreSQL              | 10.x and 11.x
+| Cassandra               | 3.11+
+| Newts                   | 1.5.2
+| Helm                    | 3+
+| Kafka                   | 2.3.0
```

Comment:
That's the version of the client library we use, but it's backwards compatible.

Reply:
@j-white Yes, I've picked it up from the pom.xml. Should we write 1.x - 2.x here?

Reply:
Works for me.